### PR TITLE
Add external validation to PAM

### PR DESCRIPTION
**Pull request by danielcopaciu**
Previously, PAM was doing local validation in order to make sure that the received content is valid for publishing. This is not true for methode articles where PAM was sending a request with the content to methode-article-transformer and if this returns 200 the content was marked as valid. This task is about making PAM validate images and lists with methode-lists-mapper and methode-image-model-mapper respectively and drop local validations for this type of contents.

https://github.com/Financial-Times/publish-availability-monitor/pull/117
https://github.com/Financial-Times/publish-availability-monitor/pull/118
https://github.com/Financial-Times/publish-availability-monitor/pull/120

https://github.com/Financial-Times/publish-availability-monitor/pull/124

### CODE REVIEW

**Comment by ancabusuioc** [comment]
Tested in XP.

File: publish-availability-monitor@.service [hunk 1]
```diff
@@ -15,8 +15,10 @@ ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p-
 ExecStartPre=/bin/bash -c 'docker history coco/publish-availability-monitor:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/publish-availability-monitor:$DOCKER_APP_VERSION'
 
 ExecStart=/bin/bash -c '\
-    export MAT_VALIDATION_URL=$(/usr/bin/etcdctl get /ft/config/monitoring/pam-mat-validation-url); \
+    export MAT_VALIDATION_URL=$(/usr/bin/etcdctl get /ft/config/monitoring/pam-mam-validation-url); \
```
Comment (tosan88) [inline publish-availability-monitor@.service]: shouldn't the env be renamed as well?

Comment (Nesze) [inline publish-availability-monitor@.service]: Renamed. 

(Though, I would rather not use any reference to the validation service name at all - e.g. mapper, transformer. Probably in the next PR.).

**Comment by Nesze** [comment]
For the record,
Running these on both clusters:
```
etcdctl set /ft/config/monitoring/pam-mam-validation-url "https://pre-prod-up.ft.com/__methode-article-mapper/map"
etcdctl set /ft/config/monitoring/pam-mcpm-validation-url "https://pre-prod-up.ft.com/__methode-content-placeholder-mapper/map"
etcdctl set /ft/config/monitoring/pam-mimm-validation-url "https://pre-prod-up.ft.com/__methode-image-model-mapper/map"
etcdctl set /ft/config/monitoring/pam-ml-validation-url "https://pre-prod-up.ft.com/__methode-list-mapper/map"
etcdctl rm /ft/config/monitoring/pam-mat-validation-url
```

**Pull request merged**